judges/find-all-issues/find-all-issues.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# MIT License
+#
+# Copyright (c) 2024 Zerocracy
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+require 'fbe/fb'
+require 'fbe/octo'
+require 'fbe/iterate'
+require 'fbe/if_absent'
+require 'fbe/who'
+require 'fbe/issue'
+
+Fbe.iterate do
+  as 'min-issue-was-found'
+  by "(agg (and (eq where 'github') (eq repository $repository) (eq what 'issue-was-opened')) (min issue))"
+  quota_aware
+  over do |repository, issue|
+    repo = Fbe.octo.repo_name_by_id(repository)
+    begin
+      after = Fbe.octo.issue(repo, issue)[:created_at]
+    rescue Octokit::NotFound
+      Fbe.fb.query("(and (eq where 'github') (eq repository #{repository}) (eq issue #{issue}))").delete!
```

Comment:
@Yegorov this line looks pretty dangerous. The functionality of cleaning up the factbase in case GitHub issues are gone -- is not part of this judge, I believe. It's better to ignore this situation here. We can introduce another judge for this later (feel free to submit a ticket).

Reply:
@yegor256 fixed, please check